src/Symbols/include/Symbols/MockSymbolCacheInterface.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYMBOLS_MOCK_SYMBOL_CACHE_INTERFACE_H_
+#define SYMBOLS_MOCK_SYMBOL_CACHE_INTERFACE_H_
+
+#include "Symbols/SymbolCacheInterface.h"
+
+namespace orbit_symbols {
+class MockSymbolCacheInterface : public SymbolCacheInterface {
+ public:
+  MOCK_METHOD(std::filesystem::path, GenerateCachedFileName, (const std::filesystem::path&),
```

Comment:
Also out of scope, but maybe this method should be `GenerateCachedFilePath`, assuming it returns the full path?

Reply:
Thanks for the suggestion! I will change it in a follow up PR (after the Microsoft symbol provider being merged so I won't have too much work for the rebase 🙈 )